plugins.audio.pbsfm/addon.py

```diff
@@ -0,0 +1,72 @@
+from xbmcswift2 import Plugin, xbmcgui
```

Comment:
xbmcswift2 is seriously outdated and not maintained any more, so it may break with any Kodi Python API change.